class/Defaults/Trigger/WordPress/ActivePlugin.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * WordPress Active Plugin trigger
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Defaults\Trigger\WordPress;
+
+use BracketSpace\Notification\Defaults\MergeTag;
+use BracketSpace\Notification\Abstracts;
+
+/**
+ * Active Plugin trigger class
+ */
+class ActivePlugin extends Abstracts\Trigger{
+
+	/**
+	 * Constructor
+	 */
+	public function __construct(){
+
+		parent::__construct( 'wordpress/active_plugin',  __( 'Plugin Active', 'notification' ) );
+
+		$this->add_action( 'activated_plugin', 10, 2 );
+		$this->set_group( __( 'WordPress', 'notification' ) );
+		$this->set_description( __( 'Fires when plugin is active', 'notification' ) );
+
+	}
+
+	/**
+	 * Gets specific update type
+	 *
+	 * @param  string $plugin PluginPath.
+	 * @return void
+	 */
+	public function action($plugin)
+	{
+		$this->plugin = $plugin;
+	}
+
+	/**
+	 * Registers attached merge tags
+	 *
+	 * @return void
+	 */
+	public function merge_tags()
+	{
+		$this->add_merge_tag( new MergeTag\StringTag( array(
+			'slug'        => 'plugin_name',
+			'name'        => __( 'Plugin Name', 'notification' ),
+			'description' => __( 'Name deactivated plugin.', 'notification' ),
```

Comment:
It's better to have a rendered value example here, ie. `Akismet`.
Needs another param: `'example' => true`